OmniCore.Mobile/OmniCore.Mobile.Android/Properties/AndroidManifest.xml

```diff
@@ -3,4 +3,6 @@
 	<uses-sdk android:minSdkVersion="23" android:targetSdkVersion="28" />
 	<application android:label="OmniCore" android:largeHeap="true"></application>
 	<uses-feature android:name="android.hardware.bluetooth_le" android:required="false" />
+	<uses-permission android:name="android.permission.BLUETOOTH"/>
```

Comment:
these are not necessary, xamarin framework compiles the attributes in the assembly info into the manifest

Reply:
Ok. * Please also ignore *
Did not know. Found this somewhere in the documentation.
(Was part of my "test" for "auto-toggle" BT)